Sources/HipMobileUI/Pages/UserRatingPage.xaml.cs

```diff
@@ -14,14 +14,61 @@
 
 using PaderbornUniversity.SILab.Hip.Mobile.UI.Navigation;
 using PaderbornUniversity.SILab.Hip.Mobile.UI.ViewModels.Pages;
+using Xamarin.Forms;
 
 namespace PaderbornUniversity.SILab.Hip.Mobile.UI.Pages
 {
     public partial class UserRatingPage : IViewFor<UserRatingPageViewModel>
     {
+        private double width;
+        private double height;
+
```

Comment:
There should always be a DeviceOrientation variable in these classes that reflects the current device orientation. It would be nice if you could add it here as well. (I know some classes are missing it; see comment below)

Reply:
Done

Reply:
That's true, but maybe someone in the future wants to utilize the DeviceOrientation. It's nice to have it already present in every xaml.cs file.